dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops

```diff
@@ -54,3 +54,35 @@ CWMOT5004.factory.used.without.metadata.useraction=Review your code to see why t
 CWMOT5005.mptelemetry.unknown.log.source=CWMOT5005W: The MicroProfile Telemetry logging feature does not recognize the [ {0} ] log source.
 CWMOT5005.mptelemetry.unknown.log.source.explanation=The Liberty runtime environment does not provide the specified log source.
 CWMOT5005.mptelemetry.unknown.log.source.useraction=Check the configuration information and specify one of the supported sources for MicroProfile Telemetry.
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Monitor Metrics messages
+#-----------------------------------------------------------------------------------------------------------------------------
+
+threadpool.activeThreads.description=The number of threads that are actively running tasks.
+threadpool.size.description=The size of the thread pool.
+
+session.activeSessions.description=The number of concurrently active sessions. A session is active when the runtime is processing a request that uses that user session.
+session.created.description=The number of sessions logged in since this metric was enabled.
+session.invalidated.description=The number of sessions logged out since this metric was enabled.
+session.invalidatedbyTimeout.description=The number of sessions logged out because of a timeout since this metric was enabled.
+session.live.description=The number of users that are currently logged in.
+
+connectionpool.handle.count.description=The number of connections that are in use. This number might include multiple connections that are shared from a single managed connection.
+connectionpool.connection.created.description=The total number of managed connections that were created since the pool creation.
+connectionpool.connection.destroyed.description=The total number of managed connections that were destroyed since the pool creation.
+connectionpool.connection.free.description=The number of managed connections that are available.
+connectionpool.connection.waitTime.description=The amount of time that connection requests waited for a connection.
+connectionpool.connection.useTime.description=The amount of time connections were used for.
```

Comment:
```suggestion
connectionpool.connection.useTime.description=The amount of time a connection was used for.
```

Reply:
@dmuelle  sorry, I was wrong. A datasource can have multiple connections. (datasource being a database)

Reply:
so is this the total amount of time for all connections to the configured datasource?

Reply:
Yup the time spent by connections made to the data source (The metric has a label/tag/key attribute that indicates which data source it is)

Reply:
@dmuelle

Reply:
Is it the total concurrent time between the start of the earliest connection and the end of the latest, or the sum of  each individual connection time, regardless of whether they're concurrent (ie connection 1 duration + connection 2 duration + ....connection n duration)

Reply:
For the `connectionpool.connection.useTime` it is the sum of each. It records the time of each connection .